Cargo.toml

```diff
@@ -16,7 +16,9 @@ nix = "0.17"
 dlib = "0.4"
 lazy_static = "1.0"
 memmap = "0.7"
-andrew = { version = "0.3.0", optional = true }
+raqote = "0.8"
+font-kit = "0.7"
```

Comment:
Be ware of `font-kit`, it'll start linking to system libs, also font-kit isn't something that works on linux, since it's lacking font fallback and you won't be able to naturally handle certain things.

Reply:
a) rusttype could be used directly.
b) [font-kit's FAQ](https://github.com/servo/font-kit/wiki/FAQ#ok-but-i-dont-care-about-hinting-can-i-have-a-pure-rust-implementation-of-font-loading) states it would welcome a PR that adds a rusttype backend.

Reply:
Maybe you can use fontdue here instead of font-kit?